custom-actions/batch/freeze_frame_using_mux.py

```diff
@@ -0,0 +1,66 @@
+"""Connect a MUX to a node and freeze the current frame"""
+
+def get_batch_custom_ui_actions():
+    """Return custom actions to execute on Batch objects."""
+
+    def get_context():
+        """Get the first available slot in a Batch Group's Context views list."""
+        import flame
+        contexts = flame.batch.contexts
+        for context in range(1, 11):
```

Comment:
Might be worth having constant for 1 and 11.

Reply:
Should I put them at the beginning of the file, the beginning of the get_ context function or above the for loop?